namespace.py

```diff
@@ -597,6 +598,30 @@ def check(self, val):
 # the basis for what is populated in the user context. Anything function
 # defined here will be wrapped and made available to untrusted user code.
 USERCONTEXT_WRAPPER_INFO = {
+  'ping' :
+      {'func' : openwrt_api.ping,
+       'args' : [Str(),Int()],
+       'return' : Dict()},
+  'traceroute' :
+      {'func' : openwrt_api.traceroute,
+       'args' : [Str(),Int(),Int(),Int(),Int()],
+       'return' : List()},
+  'get_network_bytes' :
+      {'func' : openwrt_api.get_network_bytes,
+       'args' : [Str()],
+       'return' : Dict()},
+  'get_network_packets' :
+      {'func' : openwrt_api.get_network_packets,
+       'args' : [Str()],
+       'return' : Dict()},
+  'get_network_interface' :
+      {'func' : openwrt_api.get_network_interface,
+       'args' : [],
+       'return' : List()},
```

Comment:
I would suggest factoring out those functions that could be implemented cross-platform and are not OpenWrt-specific. For example, `get_network_bytes` would be implemented on Mac OS X like this,
```
$ netstat -b -i en1
Name  Mtu   Network       Address            Ipkts Ierrs     Ibytes    Opkts Oerrs     Obytes  Coll
(....)
en1   1500  192.168.99    192.168.99.9     5164097     - 6811955792  3673190     - 1282089243     -
```